### PR TITLE
Daily build features: describe optional CSV logging

### PR DESCRIPTION
**Pull request by Williangalvani**
documents https://github.com/mavlink/qgroundcontrol/pull/7477

### CODE REVIEW

File: en/SettingsView/csv.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# CSV Logging
+
+> **Note** This is a daily build feature (not present in QGC v3.5)
+
+![Csv checkbox](../../assets/settings/general/csv.jpg)
+
+By checking this option, whenever a vehicle is connected and armed (unless "Save logs even if vehicle was not armed" is checked) a CSV (Comma-separated values) file will be created alongside the usual .tlog telemetry file.
+
+This CSV file contains the data from the vehicle Facts, which includes most of the relevant flight data quick analysis such as position, attitude, battery status, and others.
```
Comment (jaxxzer) [inline en/SettingsView/csv.md]:
```suggestion
This CSV file contains the most relevant vehicle telemetry data available for quick analysis such as GPS position, attitude, battery status, and others.
```

Comment (hamishwillee) [reply en/SettingsView/csv.md]: @Williangalvani I don't have permission to push to this repo for some reason. But I think you should accept this.

File: en/SettingsView/csv.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# CSV Logging
+
+> **Note** This is a daily build feature (not present in QGC v3.5)
+
+![Csv checkbox](../../assets/settings/general/csv.jpg)
+
+By checking this option, whenever a vehicle is connected and armed (unless "Save logs even if vehicle was not armed" is checked) a CSV (Comma-separated values) file will be created alongside the usual .tlog telemetry file.
```
Comment (hamishwillee) [inline en/SettingsView/csv.md]: Propose:

> When checked, a CSV (comma-separated value) telemetry file will be created along with the usual **.tlog** telemetry file.
  The file is only created if **Save log after each flight** is enabled, and is recorded for the same duration.

Is the option (and one above) disabled if the first **Save log after each flight** is disabled? If not, it should be!

Comment (Williangalvani) [reply en/SettingsView/csv.md]: Yes it is. I just updated the text with your suggestions

**Comment by hamishwillee** [comment]
Thanks very much. I merged and did a minor tidy: https://github.com/mavlink/qgc-user-guide/commit/649757a5a47c3f4c9f4c6396e4a3e8ec3a228e6b

**Comment by hamishwillee** [comment]
PS Don't know if you saw my comment, but I hope that the UI only enables the csv option if the top/tlog option is checked.

**Comment by jaxxzer** [comment]
> PS Don't know if you saw my comment, but I hope that the UI only enables the csv option if the top/tlog option is checked.

This is the way it works @hamishwillee

**Pull request merged**